client/src/Categories/CategoryExplorer.js

```diff
@@ -5,15 +5,15 @@ import {
 } from 'react-router-dom';
 import { connect } from 'react-redux';
 import { Card, Image, Loader, List } from 'semantic-ui-react';
-import fetchCategories from './actions/categoriesActions';
-import moneyImg from './images/money.png';
-import crimeImg from './images/crime.png';
-import healthImg from './images/health.png';
-import homeImg from './images/home.png';
-import immImg from './images/immigration.png';
-import schoolImg from './images/school.png';
-import workImg from './images/work.png';
-import famImg from './images/family.png';
+import { fetchCategories } from '../actions/categoriesActions';
+import moneyImg from '../images/money.png';
```

Comment:
Should be "Images" not "images" since the folder name has a capitalized "I", threw an error for me because of that.  This goes for all the pictures.